File: 백준/Silver/1065. 한수/한수.c
#include <stdio.h>
void hansoo(int N){
    if(N<100)
        printf("%d",N);
    else {
        int i;
        int count=99;
        int a,b,c;
        for(i=100;i<=N;i++){
            a=i/100;
            b=i/10%10;
            c=i%10;
            if ((c-b)==(b-a))
                count++;
        }
        printf("%d",count);
    }
}
int main(){
    int N;
    scanf("%d",&N);
    hansoo(N);
}

File: 백준/Silver/1065. 한수/한수.swift
func BOJ_1065(_ N: Int) {
    if N < 100 {
        print(N)
        return
    }
    var count = 99
    for i in 100...N {
        let a = i / 100, b = i / 10 % 10, c = i % 10
        if c-b == b-a { count += 1 }
    }
    print(count)
}
BOJ_1065(Int(readLine()!)!)